tests/test_move_and_ppr_change_resolution.py

```
@@ -0,0 +1,621 @@
+            assert by_id[14].text == "relocated clause"  # full text it moved in
+            assert by_id[14].contains_ids == (20,)
+            assert by_id[20].type == "deletion"
+            assert by_id[20].nested_under == 14
+            assert doc.accept_all() == 2
+            assert doc.get_visible_text() == "relocated "
+            assert _census(doc) == {}
+
+
+class TestDamagedFiles:
+    @pytest.mark.parametrize(
+        "body,accepted,rejected",
```

Ruff (0.16.2)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)